### PR TITLE
Added Bar Background Color and Alpha

### PR DESCRIPTION
**Pull request by fuba82**
Hi,
I added the ability to change the Bar Background and Alpha.

### CODE REVIEW

**Comment by funkydude** [comment]
Thanks. Can you please fix the tab/whitespace to match the rest of the addon and remove the comment on line 88?

**Comment by fuba82** [comment]
damn forgot to fix it, sorry
now it should be fine!
i've also formated the "colorBarBackground" section in Options to fit with your sections.

**Comment by funkydude** [comment]
I still see broken tabs in LegionInvasionTimer.lua and the removal of newlines at the end of the file in both files for no reason.

**Comment by fuba82** [comment]
ah forgot about the "colorBarBackground" check.
An the white spece at the end of the file is automaticly added by Github if edit a file online...

File: LegionInvasionTimer.lua [hunk 1]
```diff
@@ -18,6 +18,7 @@ frame:SetClampedToScreen(true)
 frame:Hide()
 frame:RegisterEvent("PLAYER_LOGIN")
 
+
```
Comment (funkydude) [inline LegionInvasionTimer.lua:21]: This space is unnecessary.

File: LegionInvasionTimer.lua [hunk 2]
```diff
@@ -163,6 +165,7 @@ frame:SetScript("OnEvent", function(f)
 			colorText = {1,1,1,1},
 			colorComplete = {0,1,0,1},
 			colorIncomplete = {1,0,0,1},
+      colorBarBackground = {0,0,0,0.75},
```
Comment (funkydude) [inline LegionInvasionTimer.lua:168]: Broken tab.

File: Options/Options.lua [hunk 3]
```diff
@@ -331,10 +331,31 @@ local acOptions = {
...
+			name = L.barBackGroundColor,
+			type = "color",
+			hasAlpha = true,
+			order = 18.1,
+			get = function()
+				return unpack(lit.db.colorBarBackground)
+			end,
+			set = function(info, r, g, b, a)
+				lit.db.colorBarBackground = {r, g, b, a}
+				if lit.bar1:Get("LegionInvasionTimer:complete") then
```
Comment (funkydude) [inline Options/Options.lua]: Is there a reason to restrict this to `LegionInvasionTimer:complete` ?

**Comment by fuba82** [comment]
yes because if there is no "bar1, bar2 or bar3" you will get an error barx is nil.
for sure you can shrink this by use "lit.bar1" i think but it mages no change at all ^^

**Comment by funkydude** [comment]
Just fix the last 2 whitespace issues and all looks good!

I assume you've tested this right? :D

**Comment by fuba82** [comment]
I use this with all the whitespaces and comments since first pull request :P
also tried to make all dynamic but its hard to do with all the subtables like lit.barX ^^

maybe you take a look at KalielsTracker's LegionInvasion.lua.
he uses a dynamic method to create the timers with:

```
local mapIDs = { 181, 11, 161, 27, 24, 39 }
for _, id in ipairs(mapIDs) do
        name, timeLeftMinutes = GetInvasionInfoByMapAreaID(id)
end
```

but your timer is more accurate ;)

**Comment by funkydude** [comment]
Both those commits seem to be empty..?
The whitespace issues still exist.

I have to clue what you mean by creating the timers "dynamic". I already take the time from the API, so your reply here is confusing.

**Comment by funkydude** [comment]
I'll just merge this and fix it since this is dragging on now, thanks for your work!

**Pull request merged**